lib/spark_api/models/search_template/quick_search.rb

```diff
@@ -5,7 +5,12 @@ class QuickSearch < Base
       include Concerns::Savable,
               Concerns::Destroyable
 
-      self.element_name="searchtemplates/quicksearches"
+      self.element_name = "searchtemplates/quicksearches"
+
+      def fields(args = {})
+        arguments = {:_expand => "Fields"}.merge(args)
```

Comment:
One thing that would be good to add to this is a check to see if the instance already has a Fields attribute so we don't issue an extra request if we already have the needed data.